django/db/migrations/writer.py

```diff
@@ -40,9 +41,10 @@ def __init__(self, value, setting_name):
 class OperationWriter(object):
     indentation = 2
```

Comment:
You can drop that line since you are no initializing `self.indention` in `__init__()`.